Jenkinsfile

```diff
@@ -74,9 +75,36 @@ pipeline{
                     else
                         cp $ML_RPM .
                     fi
+                if [-z ${env.ML_CONVERTERS}]; then
```

Comment:
@vitalykorolev  Let me know if you think there is a better way to do this, I basically copied the RPM installation logic and changed the SCP command to grab the converters as well.